changelog.md

```diff
@@ -1,3 +1,110 @@
+<a name="2.44.0"></a>
+# [2.44.0](https://github.com/iTowns/itowns/compare/v2.43.1...v2.44.0) (2024-09-02)
+
+
+### Features
+
+* **3dtiles:** add new OGC3DTilesLayer using 3d-tiles-renderer-js ([a2fedd8](https://github.com/iTowns/itowns/commit/a2fedd8))
```

Comment:
A part of the commit is missing, could you add it manually please ? (maybe in new lines ? WDYT?):
```
Deprecate C3DTilesLayer (replaced by OGC3DTilesLayer).
Add new iGLTFLoader that loads gltf 1.0 and 2.0 files.
```

Reply:
Mmmh ok I see thanks. In this case I'm all for not doing it because it's not worth breaking changelog generation for all future releases. I'll be more careful in my commits message next time.

Reply:
Okay, I removed the added lines then :)